plugins/inventory/ntnx_prism_vm_inventory.py

```diff
@@ -64,6 +64,11 @@
             type: boolean
             env:
                 - name: VALIDATE_CERTS
+        filters:
+            description: A list of Jinja2 expressions used to filter the inventory
```

Comment:
can you mention that the all filters are applied as AND expression (filter1 AND filter2 AND ..)